docs/querying/api.md

```diff
@@ -653,6 +653,52 @@ curl -G http://localhost:9091/api/v1/targets/metadata \
 }
 ```
 
+## Querying metric metadata
+
+Similarly to the previous endpoint `api/v1/targets/metadata`, it returns metadata about metrics currently scrapped by targets. However, it does not provide any target information.
+This is considered **experimental** and might change in the future.
+
+```
+GET /api/v1/metadata
+```
+
+URL query parameters:
+
+- `limit=<number>`: Maximum number of metrics to return.
+
+The `data` section of the query result consists of an object where the keys represent a metric name and the values are a list of the non-matching metadata found on the scraped targets. The metadata that is equal across all targets is collapsed, and aggregates under the same metric name the ones that differ.
+
+The following example returns two metrics, notice that the metric `http_requests_total` has a more than one object in the list. At least one target had a value for `HELP` that did not match with the rest.
```

Comment:
"…metrics, notice…" → …metrics. Note…"

I would not use the past tense in the 2nd sentence. (This API is only returning the metadata from active targets.)

Reply:
The targets are indeed active but the action (scrape) is in the past, considering we only output data that was scraped beforehand.

I'll correct it as I still think it reads better even if is not _as_ accurate.